pcsx2/pcsx2.vcxproj

```diff
@@ -63,6 +63,7 @@
       <!-- SH 2/3 flashlight, explicitly set here don't change -->
       <!-- https://github.com/PCSX2/pcsx2/commit/16431653e4d92fda4069031897e24fbe4688d36a -->
       <FloatingPointModel>Precise</FloatingPointModel>
+      <RuntimeLibrary Condition="'$(Configuration)|$(Platform)'=='Release AVX2|x64'">MultiThreadedDLL</RuntimeLibrary>
```

Comment:
Unrelated change?

Reply:
hmmm...for these VS project changes they do seem unrelated but i didn't do any of them myself.  Good catch